plugins/jdbc-adapter-framework/src/main/java/org/polypheny/db/adapter/jdbc/stores/AbstractJdbcStore.java

```diff
@@ -492,10 +492,6 @@ public void rollback( PolyXid xid ) {
     @Override
     public void shutdown() {
         try {
-            if ( deployMode == DeployMode.DOCKER ) {
-                // This call is supposed to destroy all containers belonging to this adapterId
-                DockerContainer.getContainerByUUID( deploymentId ).ifPresent( DockerContainer::destroy );
-            }
```

Comment:
Why removing this?

Reply:
Because it is redundant.  The `Adapter` takes care of this (see [here](https://github.com/polypheny/Polypheny-DB/blob/various-improvements/core/src/main/java/org/polypheny/db/adapter/Adapter.java#L151-L157).